bases/rsptx/interactives/runestone/activecode/activecode.py

```diff
@@ -454,6 +454,7 @@ def run(self):
             suffix = "\n"
 
         course_name = env.config.html_context["course_id"]
+        self.options["divid"] = f"activecode-{self.options.get('divid')}"
```

Comment:
I think this would be actively harmful to existing books with identifiers in our database as it would end up adding activecode in front of their ids.  The base class RunestoneIdDirective sets the divid in options.